nix/gecko_env.nix

```diff
@@ -74,7 +74,6 @@ in gecko.overrideDerivation (old: {
     # Build custom mozconfig
     mozconfig=$out/conf/mozconfig
     echo > $mozconfig "
-    ac_add_options --enable-clang-plugin
     ac_add_options --enable-debug
     ac_add_options --with-clang-path=${clang_4}/bin/clang
     ac_add_options --with-libclang-path=${llvmPackages_4.libclang}/lib
```

Comment:
Do we still need these options (--with-clang-path and --with-libclang-path) if we disable clang plugin?

Reply:
These two weren't even used for clang-plugin build but as far as I remember they were used for bindgen, so yes we can remove them as well.
But I will do a separate PR.

Reply:
Yes, separate PR is better!